workerapi/server/setup.go

```diff
@@ -59,10 +58,6 @@ func Defaults() (*ice.MagicBag, jsonconfig.Schema) {
 
 		func() (*temp.TempDir, error) { return temp.TempDirDefault() },
 
```

Comment:
Why do you need to override prior to returning from Defaults() when you can just override the returned values?